exec/java-exec/src/main/java/org/apache/drill/exec/compile/ClassBuilder.java

```diff
@@ -64,36 +67,33 @@
  * local variables. Have fun!</li>
  * </ul>
  * <p>
- * Note: not all generated code is ready to be compiled as plain-old
- * Java. Some classes omit from the template the proper <code>throws</code>
- * declarations. Other minor problems may also crop up. All are easy
- * to fix. Once you've done so, add the following to mark that you've
- * done the clean-up:<pre>
- * cg.plainOldJavaCapable(true); // Class supports plain-old Java</pre>
+ * Most generated classes have been upgraded to support Plain-old Java
```

Comment:
Hard to put that list anywhere but in the operators that have the issues (else we have to keep the comments in sync.) There are two. The old `SortBatch`. Also `OrderedPartitionRecordBatch`, which seems to not be used at present. These are called out in the JIRA.